FlowCrypt/src/androidTest/java/com/flowcrypt/email/SendMsgTest.kt

```diff
@@ -572,15 +592,33 @@ class SendMsgTest {
     )
   )
 
-  companion object {
-    const val ATTACHMENT_NAME = "test.txt"
+  private fun prepareForwardedAttachment(outgoingMessageInfo: OutgoingMessageInfo): AttachmentInfo? {
+    processOutgoingMessageInfo(outgoingMessageInfo)
+
+    val worker = TestListenableWorkerBuilder<MessagesSenderWorker>(context).build()
+    return runBlocking {
+      val result = worker.doWork()
+      assertThat(result, `is`(ListenableWorker.Result.success()))
+      val list = mutableListOf<AttachmentInfo>()
+      checkExistingMsgOnServer(
+        JavaEmailConstants.FOLDER_INBOX,
+        outgoingMessageInfo
+      ) { folder, mimeMessage ->
+        val uid = folder.getUID(mimeMessage)
+        list.addAll(EmailUtil.getAttsInfoFromPart(mimeMessage).map {
+          it.copy(
+            email = addAccountToDatabaseRule.account.email,
+            folder = folder.fullName,
+            uid = uid
+          )
+        })
+      }
 
-    //It's a real name of the first attachment of the message with Uid = 0
-    //in INBOX for default@flowcrypt.test
-    const val FORWARDED_ATTACHMENT_NAME = "flowcrypt-backup-defaultflowcrypttest.key"
+      return@runBlocking list.firstOrNull()
```

Comment:
Instead of mutable `list`, we can `checkExistingMsgOnServer` lambda to return the first attachment found.
```kt
checkExistingMsgOnServer(JavaEmailConstants.FOLDER_INBOX, outgoingMessageInfo) { folder, mimeMessage ->
    val uid = folder.getUID(mimeMessage)
    EmailUtil.getAttsInfoFromPart(mimeMessage).map {
        it.copy(email = addAccountToDatabaseRule.account.email, folder = folder.fullName, uid = uid)
    }.firstOrNull()
}
```

Reply:
It requests additional changes in the called functions. But it will provide simplifications. I will add changes.